cellengine/utils/fcs_file_io.py

```diff
@@ -0,0 +1,55 @@
+from io import BufferedReader, BytesIO
+from typing import BinaryIO, List, Optional, Union
+
+import flowio
+from flowio import FlowData, create_fcs
+from numpy import reshape
+from pandas.core.frame import DataFrame
+
+
+class FcsFileIO:
+    @classmethod
+    def parse(
+        cls, file: Union[BinaryIO, str], destination: str = ""
+    ) -> Optional[DataFrame]:
+        """Parse an FCS file to a Dataframe
```

Comment:
I'd split this up into `save` and `parse` methods. It doesn't make much sense for `parse` to save a file.